src/DiagnosticMessages.ts

```diff
@@ -24,711 +28,841 @@ export let DiagnosticMessages = {
     }),
     keywordMustBeDeclaredAtRootLevel: (keyword: string) => ({
         message: `${keyword} must be declared at the root level`,
-        code: 1056,
-        severity: DiagnosticSeverity.Error
+        legacyCode: 1056,
+        severity: DiagnosticSeverity.Error,
+        code: 'keyword-must-be-root-level'
     }),
     __unused5: () => ({
         message: `'library' statement must be declared at the top of the file`,
-        code: 1057,
+        legacyCode: 1057,
         severity: DiagnosticSeverity.Error
     }),
     expectedEndIfElseIfOrElseToTerminateThenBlock: () => ({
         message: `Expected 'end if', 'else if', or 'else' to terminate 'then' block`,
-        code: 1058,
-        severity: DiagnosticSeverity.Error
+        legacyCode: 1058,
+        severity: DiagnosticSeverity.Error,
+        code: 'expected-terminator-on-then'
     }),
     expectedEndTryToTerminateTryCatch: () => ({
         message: `Expected 'end try' to terminate 'try-catch' statement`,
-        code: 1059,
-        severity: DiagnosticSeverity.Error
+        legacyCode: 1059,
+        severity: DiagnosticSeverity.Error,
+        code: 'expected-terminator-on-try-catch'
     }),
     expectedEndIfToCloseIfStatement: (startingPosition: Position) => ({
         message: `Expected 'end if' to close 'if' statement started at ${startingPosition?.line + 1}:${startingPosition?.character + 1}`,
-        code: 1060,
-        severity: DiagnosticSeverity.Error
+        legacyCode: 1060,
+        severity: DiagnosticSeverity.Error,
+        code: 'expected-terminator-on-if'
     }),
     expectedStatementToFollowConditionalCondition: (conditionType: string) => ({
         message: `Expected a statement to follow '${conditionType?.toLowerCase()} ...condition... then'`,
-        code: 1061,
-        severity: DiagnosticSeverity.Error
+        legacyCode: 1061,
+        severity: DiagnosticSeverity.Error,
+        code: 'expected-statement-after-conditional'
     }),
     expectedStatementToFollowElse: () => ({
         message: `Expected a statement to follow 'else'`,
-        code: 1062,
-        severity: DiagnosticSeverity.Error
+        legacyCode: 1062,
+        severity: DiagnosticSeverity.Error,
+        code: 'expected-statement-after-else'
     }),
     consecutiveIncrementDecrementOperatorsAreNotAllowed: () => ({
         message: `Consecutive increment/decrement operators are not allowed`,
-        code: 1063,
-        severity: DiagnosticSeverity.Error
+        legacyCode: 1063,
+        severity: DiagnosticSeverity.Error,
+        code: 'consecutive-increment-decrement'
     }),
     incrementDecrementOperatorsAreNotAllowedAsResultOfFunctionCall: () => ({
-        message: ``,
-        code: 1064,
-        severity: DiagnosticSeverity.Error
+        message: `Increment/decrement operators are not allowed on function calls`,
+        legacyCode: 1064,
+        severity: DiagnosticSeverity.Error,
+        code: 'increment-decrement-on-function-call'
```

Comment:
Merge this diagnostic in with the previous one (1063) because they're logically the same thing